src/domains/transaction/pages/SendTransfer/SendTransfer.Fees.test.tsx

```diff
@@ -90,6 +92,14 @@ describe("SendTransfer Fee Handling", () => {
 			requestMock("https://ark-live.arkvault.io/api/transactions/fees", transactionFeesFixture),
 		);
 		await syncFees(profile);
+
+		confirmedTransactionMock = vi
+			.spyOn(useConfirmedTransactionMock, "useConfirmedTransaction")
+			.mockReturnValue(true);
```

Comment:
I'd rather add specific unit test for the `useConfirmedTransaction` hook to  identify breaking changes from SDK updates. Otherwise I'd be testing implementation details as it is not concern of `SendTransfer.Fees` test to know how it is marked as confirmed internally. What do you think?

Reply:
merging this in advance so other PRs can benefit from the fixed suites